pkg/logs/metrics/metrics.go

```diff
@@ -81,6 +81,13 @@ var (
 
 	// TlmLogsDiscardedFromSDSBuffer how many messages were dropped when waiting for an SDS configuration because the buffer is full
 	TlmLogsDiscardedFromSDSBuffer = telemetry.NewCounter("logs", "sds__dropped_from_buffer", nil, "Count of messages dropped from the buffer while waiting for an SDS configuration")
+
+	// TlmUtilization is the utilization of a component
+	TlmUtilization = telemetry.NewGauge("logs_component", "utilization", []string{"name", "instance"}, "Gauge of the utilization of a component")
+	// TlmCapacity is the capacity of a component by number of elements
+	TlmCapacity = telemetry.NewGauge("logs_component", "capacity", []string{"name", "instance"}, "Gauge of the capacity of a component by number of elements")
```

Comment:
I fear the name of this metric may be misleading. Capacity is normally understood as maximum amount something can contain or handle, while this only reflects current usage level. How about `utilization.seconds`, `utilization.payloads`, `utilization.bytes`?

Also, would it be possible to place the explanation of this measures from the PR description here?

Reply:
That's fair. 
Using your suggestions i'd make the following tweaks:

Utilization is a ratio of in_use vs idle. So maybe `utilization.ratio`? (instead of `seconds`)
And then `utilization.payloads` should probably be `utilization.items` or `utilization.messages`? 
`utilization.bytes` sounds good to me. 

WDYT?

Reply:
Ive made the above changes. LMK if you have further feedback.